src/category_theory/category/ulift.lean

```diff
@@ -45,21 +45,21 @@ variables {C : Type u₁} [category.{v₁} C]
 
 /-- The functorial version of `ulift.up`. -/
 @[simps]
-def ulift.up : C ⥤ (ulift.{u₂} C) :=
+def ulift.up_functor : C ⥤ (ulift.{u₂} C) :=
```

Comment:
It conflicts with `_root_.ulift.up` in lots of files when I tried to import this into `has_limits.lean`.
It (and the definitions using it) is never used outside of this file, so I thought it would be faster if I changed the name of this.